src/metabase/task/search_index.clj

```diff
@@ -56,7 +59,7 @@
     (log/info "Reindexing searchable entities")
     (let [timer  (u/start-timer)
           report (search/reindex!)]
-      (report->prometheus! report)
+      (report->prometheus! timer report)
       (log/infof "Done reindexing in %.0fms %s" (u/since-ms timer) (sort-by (comp - val) report))
```

Comment:
Why not calculate `since-ms` once, and pass that to `report->prometheus!` instead?

Reply:
I had to know you'd write this after the discussion of disk defragmentation. 🤣